dvc/state.py

```diff
@@ -37,7 +37,10 @@ class StateNoop(object):
     def save(self, path_info, checksum):
         pass
 
-    def get(self, path_info):
+    def get(self, path_infos):
+        return {pi: None for pi in path_infos}
+
+    def get_single(self, path_info):
```

Comment:
Agree with @Suor . Let's remove this, and use `get([info])` when we need it.

Reply:
Or make `.get()` accept both lists and single path infos.

Reply:
Or make .get() accept both lists and single path infos.